README.md

```diff
@@ -65,7 +65,7 @@ Alternatively, choose below the sample you want to review.
 
 |                 |               |
 |-----------------|---------------|
-| <img src="./3-Authorization-II/1-call-api/ReadmeFiles/topology_callapi.png" width="200"> | [**Protect and call a web API on Azure AD**](.3-Authorization-II/1-call-api) </br> Protect your web API with the **Azure AD**. Use a client application to sign-in a user, acquire an **Access Token** for your web API and call your protected Web API. |
+| <img src="./3-Authorization-II/1-call-api/ReadmeFiles/topology_callapi.png" width="200"> | [**Protect and call a web API on Azure AD**](3-Authorization-II/1-call-api) </br> Protect your web API with the **Azure AD**. Use a client application to sign-in a user, acquire an **Access Token** for your web API and call your protected Web API. |
```

Comment:
Did you mean add `./` here, as in:

`[**Protect and call a web API on Azure AD**](./3-Authorization-II/1-call-api)`

Reply:
no, just erase the dot at the link, when I try to open [the link it's broke](https://github.com/Azure-Samples/ms-identity-javascript-tutorial/blob/main/.3-Authorization-II/1-call-api) because of the dot

Reply:
You are right, that would do it too. But the issue seems to be the missing `/` here. As you can see, all the other links have it except this one. I'll prefer to go that way for consistency.

Reply:
Yeah, it's better. Can I edit the pull request and put the `./` ?

Reply:
Done! Approving..

Reply:
Nice, thank you very much! I'm really happy, this was my first pull request in a public repository, glad to help 👍🏾 😄

Reply:
cheers!